core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -71,14 +73,8 @@ oppia.controller('Dashboard', [
     $rootScope.loadingMessage = 'Loading';
     DashboardBackendApiService.fetchDashboardData().then(
       function(response) {
-        $scope.explorationsList = response.explorations_list.sort(
-          function(a, b) {
-            return (a.title === '' ? 1 :
-              b.title === '' ? -1 :
-              a.title < b.title ? -1 :
-              a.title > b.title ? 1 : 0);
-          }
-        );
+        $scope.explorationsList = (
+          sortExplorationsService.sortBy(response.explorations_list, 'title'));
```

Comment:
Changed it to default sorting by open feedback in descending order. (Though, I don't recall about any such mention)